internal/cmd/testdata/scripts/templatevars.txt

```diff
@@ -9,6 +9,9 @@ chezmoi execute-template '{{ .chezmoi.arch }}'
 chezmoi execute-template '{{ index .chezmoi.args 1 }}'
 stdout execute-template
 
+chezmoi execute-template '{{ .chezmoi.executable }}'
+stdout ^.*[\\/]bin[\\/]chezmoi(.exe)?$
```

Comment:
The `^.*` at the start of the regexp doesn't have any effect: it matches any leading characters from the start of the string, which is the same as if `^.*` was not present. I'm not sure that it's guaranteed that the executable will be in a `bin` directory, so I'd remove the `[\\/]bin[\\/]` part. Overall I think the regexp should be `chezmoi(.exe)?$`.